vuu-ui/packages/vuu-shell/src/left-nav/LeftNav.css

```diff
@@ -165,12 +165,12 @@
 
 .vuuLeftNav-drawer {
     display: flex;
-    padding: 40px 32px 0px 24px;
+    padding: 0 0 0 24px;
```

Comment:
Looking at Figma, should this be `padding: 0px 32px 0px 24px`? 

![image](https://github.com/ScottLogic/finos-vuu/assets/79100986/efdd9cac-59ee-4ba9-a2d4-5a817cbc4075)

Reply:
It looks like Figma has the scrollbar as a static fixed element on the page, as opposed to attached to an object. The real render attaches the scrollbar to the right edge of the layout list component. I couldn't get the layout list to scroll by attaching the scrollbar to the drawer. I had to reverse the padding here (of the drawer) to be on the layout list instead, to prevent this:

![image](https://github.com/ScottLogic/finos-vuu/assets/99646608/14262ff2-115a-46c9-af6c-7644dd4f2eb2)

@vferraro-scottlogic Do you know a way to apply the scrollbar to the drawer whilst still scrolling the layout list?